src/openzaak/components/autorisaties/utils.py

```diff
@@ -97,10 +97,14 @@ def send_applicatie_changed_notification(
     applicatie: Applicatie, new_version: Optional[Dict[str, Any]] = None
 ):
     from openzaak.utils import build_fake_request
+    from openzaak.utils.middleware import override_request_host
 
     viewset = ApplicatieViewSet()
     viewset.action = "update"
     if new_version is None:
         request = build_fake_request(method="put")
+        # we have to explicitly override the request host (because this is not an API request)
+        # to ensure the notification has the right URLs
+        override_request_host(request)
```

Comment:
Is it necessary? I think `build_fake_request` should already use `OPENZAAK_DOMAIN`
Maybe it could be also refactored with your `override_request_host` function

Reply:
You're right, it's not needed so I removed it